### DISCUSSION TITLE
Processing Weeks Old Files from Dumb CCTV DVR

### DISCUSSION THREAD

**Discussion by Cisor** [blakeblackshear/frigate#3661] [Ask A Question]
Hi!
I have a couple of weeks of footage from a dumb DVR. 
I want to run only person detection on the files to catch a thief who stole my bike.

I am able to load the files and view them in Frigate. Detection works well :+1: 

However, I have 336 hours of footage and loads of horsepower on the desktop CPU with a Coral TPU arriving tomorrow. 

Two questions:
1. Can I run the file playback / analysis at high speed, say 8x and have Frigate work equally well? Subject of course to hardware limitations
2. Can Frigate timestamp the detected events if I feed it the genuine time that the recording was taken?

**Comment by NickM-27** [comment]
> Can I run the file playback / analysis at high speed, say 8x and have Frigate work equally well? Subject of course to hardware limitations

You'd need to increase the `detect -> fps` to match and without a coral this will be quite difficult

> Can Frigate timestamp the detected events if I feed it the genuine time that the recording was taken?

I don't think there's a way to do that

**Comment by Cisor** [comment]
Are you sure detect fps works that way? I was under the impression that this was the number of frames of the input stream Frigate would consider per second for detection. Not a multiplier of the frame rate of the incoming stream. 
I tried this setting earlier and didn't seem to get good results. I'll try again though! 

Also, thanks for a really quick reply!

**Reply by NickM-27** [reply]
Yes, that is how it works. But if your video is 30 fps and you want to play it at 2 times speed then that is 60 fps. If frigate detect fps is set to the default 5 fps then it is going to see every 12th frame for detection. There's a lot of room to miss a person running through the frame especially since frigate requires 3 detections with a median score above the threshold to consider it a true positive and create an event.

**Reply by Cisor** [reply]
Ok, I'll go back and try again. Thank you.

So my video files are 25 fps. 
I can manipulate the PTS to make them 200 fps, x8 speed without loss of frames

So if I then set my detect fps to 40, 8x the 5 fps default, would you expect that an 8 hour file would be processed in an hour. 

Assuming that of course that my CPU / TPU are capable?
